src/holoscan_cli/common/constants.py

```diff
@@ -77,19 +77,22 @@ class SDK:
     """
 
     # Platform to architecture mappings
-    PLATFORM_MAPPINGS = {
-        Platform.IGXOrinDevIt: Arch.arm64,
-        Platform.JetsonAgxOrinDevKit: Arch.arm64,
+    PLATFORM_ARCH_MAPPINGS = {
+        Platform.Jetson: Arch.arm64,
+        Platform.IGX_iGPU: Arch.arm64,
+        Platform.IGX_dGPU: Arch.arm64,
         Platform.SBSA: Arch.arm64,
         Platform.X64Workstation: Arch.amd64,
+        Platform.x86_64: Arch.amd64,
     }
 
     # Values of all platforms supported by the Packager
     PLATFORMS = [
-        Platform.IGXOrinDevIt.value,
-        Platform.JetsonAgxOrinDevKit.value,
+        Platform.Jetson.value,
+        Platform.IGX_iGPU.value,
+        Platform.IGX_dGPU.value,
         Platform.SBSA.value,
-        Platform.X64Workstation.value,
```

Comment:
For here, is there backwards compatibility concerns?

Reply:
This value is a user-facing option for the `--platform` argument.  As long as they select `x86_64`, it will be the same as `X64Workstation`